functions/Get-DbaAgentJob.ps1

```diff
@@ -67,18 +87,22 @@ FUNCTION Get-DbaAgentJob {
 			}
 
 			$jobs = $server.JobServer.Jobs
+			
 			if ($Job) {
 				$jobs = $jobs | Where-Object Name -In $Job
 			}
 			if ($ExcludeJob) {
 				$jobs = $jobs | Where-Object Name -NotIn $ExcludeJob
 			}
-
+			if (!$IncludeDisabled -and !$job) {
```

Comment:
Looking at other commands then to follow the same pattern this should actually be changed to "-NoDisabledJobs" and then by default we return all of them.

- `Get-DbaDatabase` has NoUserDb and NoSystemDb
- `Get-DbaProcess` has NoSystemSpids
- `Get-DbaTable` has IncludeSystemDbs